### DISCUSSION TITLE
failed to wait for cluster caches to sync: timed out waiting for cache to be synced

### DISCUSSION THREAD

**Discussion by Evolt0** [cloudnative-pg/cloudnative-pg#1740] [Q&A]
it always restart ,when apply this operator 
occured by failed to wait for cluster caches to sync
![image](https://user-images.githubusercontent.com/38104576/225801440-907e56d7-a18e-4a24-895e-3658604160de.png)

**Comment by sxd** [comment]
@Evolt0 the errors above show something like the operator wasn't installed properly, how was the operator installed?
Also, can you please share the error in plain text? it's a bit annoying to read an image.

I'm turning this into a discussion since it's not an issue yet

**Comment by Evolt0** [comment]
this way "kubectl apply -f \
  https://raw.githubusercontent.com/cloudnative-pg/cloudnative-pg/release-1.19/releases/cnpg-1.19.0.yaml"

**Comment by Evolt0** [comment]
{"level":"info","ts":"2023-03-17T02:47:42Z","msg":"Wait completed, proceeding to shutdown the manager"}
{"level":"error","ts":"2023-03-17T02:47:42Z","logger":"setup","msg":"problem running manager","error":"failed to wait for cluster caches to sync: timed out waiting for cache to be synced","stacktrace":"[github.com/cloudnative-pg/cloudnative-pg/pkg/management/log.(*logger).Error\n\tpkg/management/log/log.go:127\ngithub.com/cloudnative-pg/cloudnative-pg/internal/cmd/manager/controller.RunController\n\tinternal/cmd/manager/controller/controller.go:276\ngithub.com/cloudnative-pg/cloudnative-pg/internal/cmd/manager/controller.NewCmd.func1\n\tinternal/cmd/manager/controller/cmd.go:39\ngithub.com/spf13/cobra.(*Command).execute\n\tpkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\tpkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044\ngithub.com/spf13/cobra.(*Command).Execute\n\tpkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968\nmain.main\n\tcmd/manager/main.go:64\nruntime.main\n\t/opt/hostedtoolcache/go/1.19.5/x64/src/runtime/proc.go:250](http://github.com/cloudnative-pg/cloudnative-pg/pkg/management/log.(*logger).Error/n/tpkg/management/log/log.go:127/ngithub.com/cloudnative-pg/cloudnative-pg/internal/cmd/manager/controller.RunController/n/tinternal/cmd/manager/controller/controller.go:276/ngithub.com/cloudnative-pg/cloudnative-pg/internal/cmd/manager/controller.NewCmd.func1/n/tinternal/cmd/manager/controller/cmd.go:39/ngithub.com/spf13/cobra.(*Command).execute/n/tpkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916/ngithub.com/spf13/cobra.(*Command).ExecuteC/n/tpkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044/ngithub.com/spf13/cobra.(*Command).Execute/n/tpkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968/nmain.main/n/tcmd/manager/main.go:64/nruntime.main/n/t/opt/hostedtoolcache/go/1.19.5/x64/src/runtime/proc.go:250)"}
Error: failed to wait for cluster caches to sync: timed out waiting for cache to be synced
{"level":"error","ts":"2023-03-17T02:47:42Z","msg":"error received after stop sequence was engaged","error":"leader election lost","stacktrace":"[sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).engageStopProcedure.func1\n\tpkg/mod/sigs.k8s.io/controller-runtime@v0.14.2/pkg/manager/internal.go:550](http://sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).engageStopProcedure.func1/n/tpkg/mod/sigs.k8s.io/controller-runtime@v0.14.2/pkg/manager/internal.go:550)"}

**Comment by Evolt0** [comment]
{"level":"error","ts":"2023-03-17T04:11:38Z","logger":"controller-runtime.source","msg":"if kind is a CRD, it should be installed before calling Start","kind":"PodDisruptionBudget.policy","error":"no matches for kind \"PodDisruptionBudget\" in version \"policy/v1\"","stacktrace":"sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1.1\n\tpkg/mod/sigs.k8s.io/controller-runtime@v0.14.2/pkg/source/source.go:143\nk8s.io/apimachinery/pkg/util/wait.runConditionWithCrashProtectionWithContext\n\tpkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:235\nk8s.io/apimachinery/pkg/util/wait.WaitForWithContext\n\tpkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:662\nk8s.io/apimachinery/pkg/util/wait.poll\n\tpkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:596\nk8s.io/apimachinery/pkg/util/wait.PollImmediateUntilWithContext\n\tpkg/mod/k8s.io/apimachinery@v0.26.1/pkg/util/wait/wait.go:547\nsigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1\n\tpkg/mod/sigs.k8s.io/controller-runtime@v0.14.2/pkg/source/source.go:136"}
